src/ExtensionsSample/Program.cs

```diff
@@ -66,7 +69,8 @@ public static void Main(string[] args)
                 typeof(SendGridSamples),
                 typeof(TableSamples),
                 typeof(TimerSamples),
-                typeof(WebHookSamples));
+                typeof(WebHookSamples),
+                typeof(ApiHubSamples));
```

Comment:
I'd still prefer if you left these opt in as they are now - that's not a problem, right? That's the pattern we're using for all the other extensions we've been onboarding lately: EasyTables, DocumentDB, NotificaitonHubs, etc.